.github/scripts/pipeline/install_xrt.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+RETURN_CODE=0
+DISTRO=`grep '^ID=' /etc/os-release | awk -F= '{print $2}' | tr -d '"'`
+
+cd build/Release
+if [[ $DISTRO == "centos" ]]; then
+        sudo yum install -y ./xrt_*xrt.rpm
+else
+        sudo apt install ./xrt_*xrt.deb
+fi
+
+DRIVER=`lsmod | grep -i xocl`
+if [ -z "$DRIVER" ]; then
+	echo "Drivers are NOT loaded"
+ 	RETURN_CODE=1
```

Comment:
Do not mix spaces and tabs.
At least configure your editor/IDE and GitHub in the whole team so at least everybody can see them and understand the problem.

Reply:
Hi @keryell , 
Changes in files will be made to maintain uniformity with the next commit.